sample/android/app/_BUCK

```diff
@@ -0,0 +1,55 @@
+# To learn about Buck see [Docs](https://buckbuild.com/).
```

Comment:
same as https://github.com/getsentry/sentry-react-native/pull/837/files#r422104024

Reply:
Could be an improvement but I'd like to land the working sample in first.
Android actually works already